ciao-launcher/main.go

```diff
@@ -528,6 +542,14 @@ func startLauncher() int {
 	timeoutCh := make(chan struct{})
 	signal.Notify(signalCh, syscall.SIGINT, syscall.SIGTERM, syscall.SIGHUP)
 
+	configCh := make(chan payloads.Configure)
+	go connectToServer(doneCh, statusCh, configCh)
+	clusterConfig := <-configCh
```

Comment:
hi @markdryan,  I'm still having trouble with this, as far I can see, it could be good if we only have network variables (computeNet and mgmtNet), but, in CluserConfiguration we also have the diskLimit and memLimit variables which are used by the overseer.go. So, if it is loaded in a go routine, not sure how to pass those variables to the overseer which is consuming the main.go context. Do you have a recommendation for this?,  I updated my commit by applying the rest of comments.